### ISSUE TITLE
`cabal sandbox delete` erroneously reports error

### ISSUE THREAD

**Issue by lspitzner** [haskell/cabal#3214]
current master (1.25.0.0); empty directory:

~~~~
> cabal sandbox init
Writing a default package environment file to
$FOO/cabal.sandbox.config
Creating a new sandbox at $FOO/.cabal-sandbox
> cabal sandbox delete
Deleting the sandbox located at $FOO/.cabal-sandbox
cabal: $FOO/.cabal-sandbox: does not exist
> echo $?
1
~~~~

(directory is empty again at this point.)

**Comment by 23Skidoo** [comment]
Thanks, will look into it.

**Comment by 23Skidoo** [comment]
Fixed, both in `master` and in `1.24`.

**Comment by lspitzner** [comment]
Thanks!
